app/webpack/stylesheets/application.scss

```diff
@@ -13,3 +13,4 @@ $govuk-image-url-function: frontend-image-url;
 @import "components/cookie_banner";
 @import "components/forms";
 @import "components/summary";
+@import "components/typography";
```

Comment:
It would be ideal to move this file to a directory titled 'core', i.e. `app/webpack/stylesheets/core/_typography.scss`